tests/functional/v1/test_leasing_positive.py

```diff
@@ -0,0 +1,49 @@
+import tests.functional.base as base
+
+class PositiveLeasingTests(base.EsiFunctionalBase):
```

Comment:
It looks like this class already containers Offer tests, so I'm not sure if the plan is to split all tests between PositiveLeasingTests and NegativeLeasingTests; if so those categories may be far too broad. I'd suggest just simplifying to something like 'OfferTests', 'LeaseTests', etc. Also, I'd suggest putting adjectives after the noun, so `LeasePositiveTests`.

Reply:
`LeasePositiveTests` sounds much better, I agree. If we're planning to cut owner change operations, I might just make two classes, one for positive tests and one for negative tests.